src/v2/Apps/Conversation/Routes/Conversation/index.tsx

```diff
@@ -26,14 +26,14 @@ interface ConversationRouteProps {
 
 const ConstrainedHeightFlex = styled(Flex)`
   height: calc(100vh - 145px);
-  ${media.xs`
+  ${media.sm`
     height: calc(100vh - 55px);
   `}
   & > * {
     overflow-y: scroll;
     overflow-x: hidden;
   }
-  & > .fresnel-greaterThan-xs {
+  > .fresnel-greaterThan-sm {
```

Comment:
Just as an FYI, there's the ability to use your own [container here](https://github.com/artsy/fresnel/issues/110#issuecomment-584453119). Not saying this is a better way but it might be a bit nicer than digging into implementation css.

Reply:
Good call out! We'll look at improving this in a separate PR.